library/testing/TestSDKCamera.cxx

```diff
@@ -179,8 +179,30 @@ int TestSDKCamera(int argc, char* argv[])
     return EXIT_FAILURE;
   }
 
+  // Test getAzimuth
+  f3d::angle_deg_t addAzimuth = cam.getAzimuth();
+  if (!compareDouble(addAzimuth, 0.0))
```

Comment:
I would expect this to be 90, not 0 ?

Reply:
also please put this test right after the addAzimuth test

Reply:
I changed it to 90 and added it after addAzimuth.